README.md

```diff
@@ -1,60 +1,30 @@
-# What is FAST-DNA?
-A design architecture for the web that enables real-time updating of user experiences based on changing inputs. The acronym stands for fast, adaptive, secure, timeless as a design network architecture.
 
-The majority of the effort in building component frameworks goes into the markup, accessibility, localization, and more. Let FAST-DNA do the hard work so you can focus on your design language and user experience.
+# What is FAST-DNA
+FAST-DNA is an acronym, which stands for Fast, Adaptive, Secure, and Timeless Design Network Architecture. A set of resources that can be used to accelerate building Web sites and applications.
```

Comment:
nit: remove comma after acronym,

Reply:
Actually, this sentence feels a bit weird to me. I would do something like:
"FAST-DNA, which stands for **F**ast, **A**daptive, **S**ecure, and **T**imeless **D**esign **N**etwork **A**rchitecture, is a set of resources that can be used to accelerate building Web sites and applications."

Reply:
If we did the above, I would remove "which" so that it went, "FAST-DNA stands for..."